### PR TITLE
Displaying issue status to student open/closed v2

### PR DESCRIPTION
**Pull request by dkim221**

### CODE REVIEW

File: pages/partials/question.ejs [hunk 1]
```diff
@@ -38,6 +38,17 @@
           Show/hide
         </button>
       </p>
+      <p><strong> Issue status: </strong>
+        <% if (issue.open) { %>
+        <span class="badge badge-danger" data-toggle="collapse" href="#">
```
Comment (mwest1066) [inline pages/partials/question.ejs]: I don't think the `data-toggle` and `href` attributes are doing anything here?

Comment (mwest1066) [inline pages/partials/question.ejs]: Can we use the same icon styling as the issue badges elsewhere? https://github.com/PrairieLearn/PrairieLearn/blob/master/pages/instructorIssues/instructorIssues.ejs#L168

That is, add the `fa-exclamation-circle` and `fa-check-circle` icons.

**Comment by dkim221** [comment]
I just pushed a new commit using the same icon styling as issue badges elsewhere.

**Comment by mwest1066** [comment]
It looks like you copied too much when you copied the styling. This code is now failing CI tests.

**Comment by mwest1066** [comment]
Closing this PR for now. See #1438

**Pull request closed without merging**